src/unit.ts

```diff
@@ -186,7 +188,7 @@ export async function setUnitInternalCargoMass(
   unit: Unit,
   mass: number
 ): Promise<void> {
-  const lua = `return trigger.action.setUnitInternalCargo("${unit.name}", ${mass})`
+  const lua = `return trigger.action.setUnitInternalCargo("${unit.uuid}", ${mass})` // use of uuid instead of unit.name
```

Comment:
setUnitInternalCargo takes the player name, so this shouldn't change to uuid. I Think we need a computed name for the unit here like for staticObject(s)  
https://wiki.hoggitworld.com/view/DCS_func_setUnitInternalCargo

Reply:
Ok. I replaced `unit.uuid` by `unit.name`.